libs/boost/Makefile

```diff
@@ -11,13 +11,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=boost
-PKG_VERSION:=1.75.0
-PKG_SOURCE_VERSION:=1_75_0
-PKG_RELEASE:=3
+PKG_VERSION:=1.76.0
+PKG_SOURCE_VERSION:=1_76_0
+PKG_RELEASE:=1
```

Comment:
This can be replaced with AUTORELEASE to avoid changing it.

Reply:
@neheb Fixed in 3dceca8

Reply:
The commit title may need to be modified.
https://github.com/openwrt/packages/issues/14537

Reply:
Commit updated in 7109e2e

I have to be honest. It's not something very helpful and ends up sacrificing space in the commit title